src/Toolbar.js

```diff
@@ -84,4 +84,8 @@ L.Toolbar = L.Class.extend({
 
 		return depth;
 	}
-});
\ No newline at end of file
+});
+
+L.toolbar = function(options) {
```

Comment:
`L.Toolbar` is an [abstract class](https://github.com/Leaflet/Leaflet.toolbar/wiki/Building-custom-toolbars#ltoolbar) meant to serve as a base class for `L.Toolbar.Control` and `L.Toolbar.Popup`. It should not be instantiated, so we shouldn't provide a factory function for it.